argus/config/ci.py

```diff
@@ -82,6 +82,15 @@ def __init__(self, config):
                             " of a failed command."),
             cfg.BoolOpt("log_each_scenario", default=False,
                         help="Create individual log files for each scenario."),
+            cfg.StrOpt(
+                "msi_web_location",
```

Comment:
I think `msi_url`/`msi_location` or something similar, since it's not necessarily from the web